### PR TITLE
Currency

### PR DESCRIPTION
**Pull request by bandvov**

### CODE REVIEW

File: __tests__/product/product.query.test.js [hunk 1]
```diff
@@ -15,11 +15,15 @@ describe('Product queries', () => {
...
             }
+            ... on Error {
+              message
+            }
           }
         }
       `,
       variables: { product: newProduct },
     });
+    console.log(createProduct);
```
Comment (Alan18081) [inline __tests__/product/product.query.test.js:26]: Remove useless `console.log`

File: currency.worker.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  const agenda = new Agenda({ db: { address: process.env.MONGO_URL } });
+  agenda.define('set currency to database', async job => {
+    try {
+      const currency = await axios.get(process.env.CURRENCY_API_URL);
+
+      const currencyRate = {
+        lastUpdatedDate: Date.now(),
+        convertOptions: [
+          {
+            name: currencyService.currencyTypes[0],
```
Comment (Alan18081) [inline currency.worker.js:16]: It is a wrong approach, use an object with keys instead, like `CurrencyTypes.USD`, also, please, export this as a separate variable from the `currency.service.js`, not as a part of the same service

File: currency.worker.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            exchangeRate: currency.data.rates.UAH,
+          },
+          {
+            name: currencyService.currencyTypes[1],
```
Comment (Alan18081) [inline currency.worker.js:20]: It is a wrong approach, use an object with keys instead, like `CurrencyTypes.USD`, also, please, export this as a separate variable from the `currency.service.js`, not as a part of the same service

File: __tests__/product/product.query.test.js (hunk 1 above)
Comment (bandvov) [reply __tests__/product/product.query.test.js:26]: fixed

File: currency.worker.js (hunk 2 above)
Comment (bandvov) [reply currency.worker.js]: fixed

File: currency.worker.js (hunk 3 above)
Comment (bandvov) [reply currency.worker.js]: fixed

**Review by yaroslavbilokin — APPROVED** [review]

**Pull request closed without merging**